src/sage_setup/autogen/interpreters/__main__.py

```diff
@@ -1,9 +1,18 @@
-from __future__ import print_function, absolute_import
+from __future__ import absolute_import, print_function
 
+import argparse
 import os
-from sage.env import SAGE_SRC
 
 from . import rebuild
 
+parser = argparse.ArgumentParser()
+parser.add_argument("output_dir")
```

Comment:
add a help string?

Reply:
I don't know what to write that would be more explanatory than "output directory"